src/components/bottom-sheet/BottomSheetHeader.tsx

```diff
@@ -0,0 +1,106 @@
+import {LayoutChangeEvent, TouchableOpacity, View} from 'react-native';
+import React, {useState} from 'react';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {ThemeText} from '@atb/components/text';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import {BottomSheetTexts, useTranslation} from '@atb/translations';
+import {Close} from '@atb/assets/svg/mono-icons/actions';
+
+type BottomSheetHeaderWithoutNavigationProps = {
+  title?: string;
+  closeBottomSheet?: () => void;
+};
+
+// Define the interface for the size
+interface Size {
+  width: number;
+  height: number;
+}
+
+export const BottomSheetHeader = ({
+  title,
+  closeBottomSheet,
+}: BottomSheetHeaderWithoutNavigationProps) => {
+  const styles = useStyle();
+  const {t} = useTranslation();
+
+  const {theme} = useTheme();
+  const {close: closeBottomSheetDefault, onOpenFocusRef} = useBottomSheet();
+
+  const handleClose = closeBottomSheet ?? closeBottomSheetDefault;
```

Comment:
I think we can just use the `close` function from the `useBottomSheet` here, not maing it possible to override from the outside? I think the close button should have no side-effects, it should be the same as closing by pressing the backdrop?

Reply:
Yes I agree, however the `useUpdateBottomSheetWhenSelectedEntityChanges` has a function that needs to run when any of the bottom sheets in the map is to be closed. I could try to change it to a onClose instead, so there can be added code to be ran when the close button is clicked.